Dockerfile

```diff
@@ -5,13 +5,12 @@ RUN mkdir -p /data/db2 \
     && chown -R mongodb:mongodb /data/db2
 
 COPY data/ /temp/
+COPY scripts/populate-search-fields.js /temp/
```

Comment:
Putting the script into a distinct directory (`pre-processor-scripts`) at the code root.